pages/common/zip2john.md

```diff
@@ -0,0 +1,17 @@
+# zip2john
+
+> A tool to extract password hashes from zip files for use with John the Ripper password cracker.
+> This is a utility tool usually installed as part of the John the Ripper installation.
+> More information: <https://www.openwall.com/john/>.
+
+- Extract the password hash from an archive, listing all files in the archive:
+
+`zip2john {{path/to/file.zip}}`
+
+- Extract the password hash using [o]nly the specified file within the archive:
+
+`zip2john -o {{compressed_file_in_zip}} {{path/to/file.zip}}`
```

Comment:
Perhaps this better
```suggestion
`zip2john -o {{compressed_filename}} {{path/to/file.zip}}`
```

Reply:
I agree it seems less than ideal.  Since archives can contain a folder structure I didn't want to confuse "path" with the local path and wanted to point out it's the file within the archive folder structure.  Perhaps `{{archive/path/to/file}}`?  I welcome suggestions to make it clearer.

Reply:
Based on `zgrep` page
```suggestion
`zip2john -o {{path/to/compressed_file}} {{path/to/file.zip}}`
```